Package.resolved

```diff
@@ -32,8 +32,8 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/apple/swift-argument-parser.git",
       "state" : {
-        "revision" : "41982a3656a71c768319979febd796c6fd111d5c",
-        "version" : "1.5.0"
+        "revision" : "0fbc8848e389af3bb55c182bc19ca9d5dc2f255b",
+        "version" : "1.4.0"
```

Comment:
I'm not sure why this downgrade happened

Reply:
Looks like `swift build` wants this change, but I'm reverting it as it has nothing to do with the PR.